src/gui/qgsmaplayerstylecategoriesmodel.cpp

```diff
@@ -230,7 +227,7 @@ QVariant QgsMapLayerStyleCategoriesModel::data( const QModelIndex &index, int ro
         case Qt::DisplayRole:
           return tr( "Custom Properties" );
         case Qt::ToolTipRole:
-          return QVariant();
+          return tr( "Custom properties (often used by plugins and custom python code)" );
```

Comment:
According to the issue report, should't it be mentioned that the variables (and what else? are among the layer's custom properties?